index.src.html

```diff
@@ -3415,29 +3417,39 @@
   VideoFrame(data, init)
 </dfn>
 1. If |init| is not a [=valid VideoFrameBufferInit=], throw a {{TypeError}}.
-2. Let |defaultRect| be «[ "x:" → `0`, "y" → `0`,
+1. Let |defaultRect| be «[ "x:" → `0`, "y" → `0`,
```

Comment:
Numbers look all wrong?

Reply:
It's actually an easier way format lists in markdown

https://carpentries-incubator.github.io/markdown-intro/06-lists/index.html

Reply:
Huh neat, TIL.